lib/cmd/compile/scripts.js

```diff
@@ -78,6 +79,7 @@ function buildKiln() {
 function copyClientInit() {
   return h(gulp.src(path.join(__dirname, '_client-init.js'))
     .pipe(changed(destPath, { hasChanged: helpers.hasChanged }))
+    .pipe(replace('#NODE_ENV#', process.env.NODE_ENV || ''))
```

Comment:
Hey @nelsonpecora can you validate for my assumption that the `changed` plugin will recognize this file was updated in the `claycli` npm module when we publish this, so that this will automatically work for people going forward without them having to blow out `/public`?

Reply:
yes!

Reply:
oh wait I'm not sure if gulp-replace does something to futz with the cnames (I know some gulp processes do)

Reply:
Just validated that it picked up my latest change (moving the window.process stuff above mounting) without requiring a `rm -rf public` step so I think we're in business!